tests/irdl/test_attribute_definition.py

```diff
@@ -642,7 +642,7 @@ def test_informative_constraint():
     constr = MessageConstraint(NoneAttr(), "User-enlightening message.")
     with pytest.raises(
         VerifyException,
-        match="User-enlightening message.\nUnderlying verification failure: Expected attribute #none but got #builtin.int<1>",
+        match="User-enlightening message.\nUnderlying verification failure: Expected attribute none but got #builtin.int<1>",
```

Comment:
as far as I know, this is the only actual functionality change. I can revert it if it is incorrect

Reply:
Nah this looks what it should be!